posthog/api/test/test_feature_flag.py

```diff
@@ -352,6 +453,138 @@ def test_deleting_feature_flag(self):
             },
         )
 
+        # fails because /api/projects/X/feature_flags/instance_id 404s before the /history path part is taken into account
```

Comment:
Could it be valid to expect this behaviour? And test for it explicitly? Given that you wouldn't be able to get to it anyhow in the UI? Perhaps in the future we handle soft deletes but for now...

Reply:
@clarkus what do you think?

This is a case where the API is set-up to GET from `/feature_flags/{the_flag_id}/history` but the library we use to create our routes is helping out when that flag has been deleted and returning 404 before it can get to our history handler

Would we want to have a view in the UI where we can see deleted things? Or are we happy with only seeing history for things that aren't deleted? Do we want to push that into the future?